node/router/messages/src/helpers/codec.rs

```diff
@@ -24,7 +24,8 @@ use tokio_util::codec::{Decoder, Encoder, LengthDelimitedCodec};
 const MAXIMUM_HANDSHAKE_MESSAGE_SIZE: usize = 1024 * 1024; // 1 MiB
 
 /// The maximum size of a message that can be transmitted in the network.
-pub(crate) const MAXIMUM_MESSAGE_SIZE: usize = 128 * 1024 * 1024; // 128 MiB
+// TODO: with message chunking in place, it can be greatly reduced.
```

Comment:
Would you be open to already making this message/event-specific in this PR? For the ones with a compile-time `sizeof` you could already set it lower. The other ones will need more work which is definitely out of scope.